help/sources/tutorials/api/create/analytics/mixpanel.md

```diff
@@ -0,0 +1,1072 @@
+| Property | Description |
+| --- | --- |
+| `name` | The name of your base connection. Ensure that the name of your base connection is descriptive as you can use this to look up information on your base connection. |
+| `description` | An optional value that you can include to provide more information on your base connection. |
+| `connectionSpec.id` | The connection specification ID of your source. This ID can be retrieved after your source is registered and approved through the [!DNL Flow Service] API. |
+| `auth.specName` | The authentication type that you are using to authenticate your source to Platform. |
+| `auth.params.projectId` | Your Mixpanel Project Id. |
+| `auth.params.timezone` | Timezone that is defined within your Mixpanel project corresponding to the `projectId`. |
+
+**Response**
+
+A successful response returns the newly created base connection, including its unique connection identifier (`id`). This ID is required to explore your source's file structure and contents in the next step.
+
+```json
+{
+     "id": "70383d02-2777-4be7-a309-9dd6eea1b46d",
+     "etag": "\"d64c8298-add4-4667-9a49-28195b2e2a84\""
+}
+```
+
+### Explore your source {#explore}
+
+Using the base connection ID you generated in the previous step, you can explore files and directories by performing GET requests.
+Use the following calls to find the path of the file you wish to bring into [!DNL Platform]:
+
+**API format**
+
+```http
+GET /connections/{BASE_CONNECTION_ID}/explore?objectType=rest&object={OBJECT}&fileType={FILE_TYPE}&preview={PREVIEW}&sourceParams={SOURCE_PARAMS}
+```
+
+When performing GET requests to explore your source's file structure and contents, you must include the query parameters that are listed in the table below:
+
+
+| Parameter | Description |
+| --------- | ----------- |
+| `{BASE_CONNECTION_ID}` | The base connection ID generated in the previous step. |
+| `objectType=rest` | The type of object that you wish to explore. Currently, this value is always set to `rest`. |
+| `{OBJECT}` | This parameter is required only when viewing a specific directory. Its value represents the path of the directory you wish to explore. |
+| `fileType=json` | The file type of the file you want to bring to Platform. Currently, `json` is the only supported file type. |
+| `{PREVIEW}` | A boolean value that defines whether the contents of the connection supports preview. |
+| `{SOURCE_PARAMS}` | Defines parameters for the source file you want to bring to Platform. To retrieve the accepted format-type for `{SOURCE_PARAMS}`, you must encode the entire `list_id` string in base64. In the example below, `"list_id": "10c097ca71"` encoded in base64 equates to `eyJsaXN0SWQiOiIxMGMwOTdjYTcxIn0=`. |
+
+
+**Request**
+
+```shell
+curl -X GET \
+    'https://platform.adobe.io/data/foundation/flowservice/connections/70383d02-2777-4be7-a309-9dd6eea1b46d/explore?objectType=rest&object=json&fileType=json&preview=true&sourceParams=eyJsaXN0SWQiOiIxMGMwOTdjYTcxIn0=' \
```

Comment:
I think the values we have here for `list_id` and `source_params` are from Mailchimp Members. Can we update these to the Mixpanel ones? Values can just be example, but want to make sure the param for `list_id` is updated to the appropriate verbiage... unless it is also called `list_id` for Mixpanel?